developer/src/tike/main/UfrmMain.pas

```diff
@@ -1159,8 +1157,16 @@ function TfrmKeymanDeveloper.OpenFile(FFileName: string; FCloseNewFile: Boolean)
 begin
   if not IsGlobalProjectUIReady then
   begin
+    // TODO: we need to open the parent folder as a project, if possible
     // This can happen if we get a file opened via Explorer.
-    modActionsMain.NewProject(ptKeyboard);
+    ShowMessage('TODO -- open parent folder as project');
+    Exit(nil);
+  end;
+
+  if DirectoryExists(FFileName) then
+  begin
+    // This is an attempt to open a project folder?
+    // TODO
```

Comment:
Does this get a warning message?

Reply:
No, it's a TODO to finish the functionality; captured in #9948.